orchestra/utils/task_properties.py

```diff
@@ -31,11 +32,17 @@ def current_assignment(task):
             The in-progress assignment for `task`.
     """
     assignments = assignment_history(task)
-    if task.status == Task.Status.POST_REVIEW_PROCESSING:
-        # Get second-to-last assignment, since reviewer rejected
-        return assignments.reverse()[1]
-    else:
+    processing = task.assignments.filter(
```

Comment:
This function didn't support multiple successive rejections down the review hierarchy.